franklin/builder/migrations/0008_auto_20151019_1833.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('builder', '0007_auto_20151012_1951'),
+    ]
+
+    operations = [
+        migrations.DeleteModel(
+            name='Site',
+        ),
+    ]
```

Comment:
@JulianGindi As far as I can tell, this is the only way to get rid of the `id` on the site model. Since we created our own `id` based on UUID, Django tried to convert that back to the vanilla int `id` it uses for tables by default if you remove the UUID version. This of course fails. 

I couldn't think of any other way to resolve this problem.